### PR TITLE
add h87a to via

### PR DESCRIPTION
**Pull request by hineybush**
Hi

### CODE REVIEW

File: src/hineybush/h87a/h87a.json [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+        "5,8",
+        {
+          "x": 2,
+          "c": "#777777",
+          "w": 1.25,
+          "h": 2,
+          "w2": 1.5,
+          "h2": 1,
+          "x2": -0.25
+        },
```
Comment (Wilba6582) [inline src/hineybush/h87a/h87a.json]: I like what you did here, very cool

**Comment by olivia** [comment]
Hi yourself ❤️

File: src/hineybush/h87a/h87a.json (hunk 1 above)
Comment (hineybush) [reply src/hineybush/h87a/h87a.json]: idk what i did here, very cool

Comment (Wilba6582) [reply src/hineybush/h87a/h87a.json]: it's the ISO Enter

**Pull request merged**